query-helpers/acs.js

```diff
@@ -138,10 +138,15 @@ const acsProfileSQL = (ids, isPrevious = false) => `
         LOWER(category),
         '[^A-Za-z0-9]', '_', 'g'
       ) AS category,
-      --- profile ---
-      'acs' AS profile
+      --- domain ---
+      REGEXP_REPLACE(
+        LOWER(domain),
+        '[^A-Za-z0-9]', '_', 'g'
+      ) AS domain,
+      --- survey ---
+      'acs' AS survey
     FROM enriched_profile
-    GROUP BY variable, variablename, base, category, profile
+    GROUP BY variable, variablename, base, category, domain
```

Comment:
According to the old SQL, here `survey` should be included in this `GROUP BY` arguments list. Do you know why we should only include `domain` instead here?

Reply:
Why do you think `survey` should be included? In this case `profile` is analogous to `domain`, not `survey`. Also, in this SQL `survey` is hard coded as `acs` so I'm not sure what including it in a group by would accomplish (begs the question if we even need to add that `survey` field at all).

Reply:
It looked like to me that profile is analogous to survey here, not domain. Since profile used to hold the value 'acs', and now survey holds the value 'acs'.